src/google-calendar-api.js

```diff
@@ -3,28 +3,38 @@ var google = require('googleapis');
 var calendar = google.calendar('v3').events;
 var moment = require('moment');
 
-var credentials = new Promise((resolve, reject) => {
-	const authFactory = new GoogleAuth();
-	const jwtClient = new authFactory.JWT(
-		process.env.GOOGLE_CLIENT_EMAIL,
-		null,
-		process.env.GOOGLE_PRIVATE_KEY,
-		['https://www.googleapis.com/auth/calendar']
-	);
+var credentials = (function() {
```

Comment:
This module is really concerned with dealing with the Google Calendar API only. This block deals with authorizing for _any_ Google API. To me, it would improve readability and testing not only of that module but this one as well, as the `credentials` object could more easily be mocked.